metrics/package.go

```diff
@@ -14,15 +14,29 @@ func Mark(name string) {
 
 // TimeSince increases the timer metric with the given name by the time since the given time
 func TimeSince(name string, since time.Time) {
+	metrics.DefaultRegistry.Register(name, newTimerWithSmallerSampleSize)
```

Comment:
Nope, I checked it specifically (though I will admit it is a bit implicit behaviour). go-metrics returns a `DuplicateMetric` if the name already exists: https://github.com/rcrowley/go-metrics/blob/master/registry.go#L130